app/uk/gov/hmrc/helptosave/models/HTSEvent.scala

```diff
@@ -154,3 +154,16 @@ object AccountCreated {
     implicit val format: Format[ManuallyEnteredDetails] = Json.format[ManuallyEnteredDetails]
   }
 }
+
+case class GetAccountResult(nino: String, account: JsValue)
+
+object GetAccountResult {
+  implicit val format: Format[GetAccountResult] = Json.format[GetAccountResult]
+}
+
+case class GetAccountResultEvent(getAccountResult: GetAccountResult, path: String)(implicit hc: HeaderCarrier, appConfig: AppConfig) extends HTSEvent {
+  val value: ExtendedDataEvent = {
+    HTSEvent(appConfig.appName, "GetAccountResult", Json.toJson(getAccountResult), "GetAccountResult", path)
```

Comment:
to keep the transaction name consistent with the others, could we have "get-account-result" for the transaction name here?

Reply:
fixed